Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:8.9.4
+FROM node:8.11.2
 
-ARG METEOR_VERSION=1.6.1
+ARG METEOR_VERSION=1.7.0.1
```

Comment:
@impactmass how do you feel about removing the `ARG` and replacing with an `ENV`. The image would be as "flexible" but it could remove confusion about how the Meteor version is set - making it clear that the ARG won't work in downstream projects.

(We don't have to do make the change as part of this PR. Floating the idea because we've seen team members get tripped up with this, and Meteor's auto-update adds to the confusion.)

Reply:
Good thought 👍